### DISCUSSION TITLE
Introduction to Committing with Conventional Commit | Leo's dev blog – stories, insights, and ideas

### DISCUSSION THREAD

**Discussion by giscus** [hta218/leohuynh.dev#70] [Announcements]
# Introduction to Committing with Conventional Commit | Leo's dev blog – stories, insights, and ideas

Conventional commit is a specific form of committing which gives software developers a uniform system for organizing and describing their changes, making it easier to keep track of updates

https://www.leohuynh.dev/blog/introduction-to-committing-with-conventional-commit

**Comment by hta218** [comment]
#### Hey, thanks for checking out this post! 💡

If you found it useful (or have some ideas for improvement), I’d love to hear from you! Leave a comment below with your thoughts, or give a reaction to show your support. It means a lot to me!

Cheers 🍺
